docs/source/developer_guides/lora.md

```diff
@@ -369,6 +369,22 @@ output = peft_model.generate(**inputs, adapter_names=adapter_names, max_new_toke
 
 Note that the order does not matter here, i.e. the samples in the batch don't need to be grouped by adapter as in the example above. We just need to ensure that the `adapter_names` argument is aligned correctly with the samples.
 
+In certain scenarios, it is necessary to replicate specific neural network layers for each set of LoRA weights. A common use case is in classification tasks, where the classification head may need to be customized for each LoRA weight. The `modules_to_save` feature allows for the creation of multiple replicas of a given layer, each corresponding to a different set of LoRA weights. For example:
```

Comment:
I think this paragraph is not super helpful as is, since it basically just explains what `modules_to_save` is for, the relation to what was described previously is not clear. I think what would be more helpful is to just mention that the feature described above also works with `modules_to_save`. A code example is not even necessary IMO.